sdk/samples/iot/README.md

```diff
@@ -129,21 +131,27 @@ To run the samples, ensure you have the following programs and tools installed o
     </p>
     </details>
 
-- Have the latest version of [CMake](https://cmake.org/download) installed. On Linux, this installation may also take an extended amount of time (~20-30 minutes).
+- Have CMake installed.
 
     <details><summary><i>Instructions:</i></summary>
     <p>
 
-    Linux:
+    Linux (Ubuntu 18.04 or 20.04):
 
     ```bash
-    sudo apt-get purge cmake
-    sudo tar -xvzf cmake-<latest-version>.tar.gz
-    cd cmake-<latest-version>
-    ./bootstrap && make && sudo make install
+    sudo apt-get install cmake
```

Comment:
I'm not seeing `sudo apt-get update` above the `install` step which is best practice to have.

Reply:
It is on line 71, uner "- Have the following build environment setup:"  if they are following the instructions in order.  I didn't want to have to rewrite for every separate apt-get install step, but I can.  It just seems redundant.